src/08-accessing.js

```diff
@@ -103,7 +114,9 @@ function getFirstPhoneNumber(person) {}
  *  getLastPhoneNumber(person);
  *  //> 7185558611
  */
-function getLastPhoneNumber(person) {}
+function getLastPhoneNumber(person) {
+  return person.numbers[length-1];
```

Comment:
it looks like there some confusion with the length property. When using it you want to make sure you are calling it on whatever `string` or `array` you are attempting to get the length one. 

One way to think about it is that `length-1` does not evaluate to anything. 
What array do you want the length of?  - in this case person.numbers
so `person.numbers.length-1` would evaluate to the last index in the array 
therefore you want `person.numbers[person.numbers.length-1]`